### PR TITLE
improvement(nemesis dashboard): dashboard should contain the test name

### PR DESCRIPTION
**Pull request by juliayakovlev**
Add test name to the dashboard tag and dashboard name

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -4464,8 +4464,27 @@ def sct_dashboard_json_file(self):
...
+    @staticmethod
+    def sct_dashboard_parameters_for_update():
+        update_params = {"$test_name": get_test_name()}
+        return update_params
+
+    @staticmethod
+    def sct_dashboard_json_file_content_update(update_params: dict, json_file: str):
+        # Read json data to the string
+        with open(json_file, 'r') as file:
+            json_data = json.dumps(json.load(file))
```
Comment (bentsi) [inline sdcm/cluster.py:4480]: what's happening here? why load then dump?

Comment (juliayakovlev) [reply sdcm/cluster.py:4480]: `json.dumps` convert json object to string

File: sdcm/utils/common.py [hunk 2]
```diff
@@ -142,6 +142,11 @@ def get_job_name():
     return os.environ.get('JOB_NAME', 'local_run')
 
 
+def get_test_name():
+    job_name = get_job_name()
```
Comment (bentsi) [inline sdcm/utils/common.py:146]: if there is not job name, try `SCT_CONFIG_FILES` environment variable

File: sdcm/cluster.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/cluster.py]: changed

File: sdcm/utils/common.py (hunk 2 above)
Comment (juliayakovlev) [reply sdcm/utils/common.py:146]: done

**Review by bentsi — APPROVED** [review]

**Pull request merged**